src/sentry/api/endpoints/assistant.py

```diff
@@ -10,15 +10,14 @@
 from sentry.api.base import Endpoint
 from sentry.assistant import manager
 from sentry.models import AssistantActivity
-from sentry.utils.compat import zip
 
 VALID_STATUSES = frozenset(("viewed", "dismissed"))
 
 
 class AssistantSerializer(serializers.Serializer):
     guide = serializers.CharField(required=False)
     guide_id = serializers.IntegerField(required=False)
-    status = serializers.ChoiceField(choices=zip(VALID_STATUSES, VALID_STATUSES))
+    status = serializers.ChoiceField(choices=list(zip(VALID_STATUSES, VALID_STATUSES)))
```

Comment:
VALID_STATUSES is a set so k->v mapping is unstable, right? or does this new ordered-ness of dicts also apply to sets?